public/app/features/alerting/unified/api/grafana.ts

```diff
@@ -1,14 +1,10 @@
 import { lastValueFrom } from 'rxjs';
 
 import { getBackendSrv } from '@grafana/runtime';
-import { ContactPointsState, NotifierDTO, ReceiversStateDTO, ReceiverState } from 'app/types';
+import { ContactPointsState, ReceiversStateDTO, ReceiverState } from 'app/types';
 
 import { getDatasourceAPIUid } from '../utils/datasource';
 
-export function fetchNotifiers(): Promise<NotifierDTO[]> {
```

Comment:
This wasn't actually called, was already unused from the previous moves to simplified routing, I think